makefile

```diff
@@ -0,0 +1,34 @@
+.PHONY: install-devel test lint pytest check-clean-tree fix force-fix venv
+
+# Customize your venv name by running make as "ZT_VENV=my_venv_name make <command>"
+
+BASEPYTHON?=python3
+ZT_VENV?=zt_venv
+
+VENV_ACTIVATE=. $(ZT_VENV)/bin/activate
+PYTHON=${ZT_VENV}/bin/$(BASEPYTHON)
+
+SOURCES = zulipterminal tests setup.py
+
+# Default target at top
+install-devel: venv
+
+### LINT/TEST FILES ###
+
+test: lint pytest
+
+lint: venv
+	@tools/lint-all
+
+pytest: venv
+	@pytest
+
+### VENV SETUP ###
+# Short name for file dependency
+venv: $(ZT_VENV)/bin/activate
+
+# If setup.py is updated or activate script doesn't exist, update virtual env
```

Comment:
We should perhaps include the fact that the `makefile` will continue to make a new `venv` even if a _custom_ `venv` exists.

Reply:
You mean if you don't use a custom environment name for all `make` invocations, it will build the standard one?

Reply:
Yes, it seems to build the standard one without that.

Reply:
On second thought, since the custom one is a bit awkward to run and manage, perhaps we should only promote the standard one.